app/models/affiliate.rb

```diff
@@ -436,6 +454,20 @@ def validate_font_family(hash)
     errors.add(:base, 'Font family selection is invalid') if hash['font_family'].present? && !FontFamily.valid?(hash['font_family'])
   end
 
+  def validate_visual_design_json
+    return if visual_design_json.blank?
```

Comment:
Is this method needed? Can this line be moved to `#validate_visual_design_font_family`?

Reply:
@ddzz `visual_design_json` will grow to include more than just these initial fonts (it will hold color settings, etc.) so I've implemented this to be extensible and grow as the amount of data set in that object grows.